package.json

```diff
@@ -59,23 +59,22 @@
     "@storybook/addon-essentials": "^6.3.8",
     "@storybook/addon-links": "^6.3.8",
     "@storybook/react": "^6.3.8",
-    "@testing-library/dom": "^6.1.0",
-    "@testing-library/jest-dom": "^4.0.0",
-    "@testing-library/react": "^9.1.3",
-    "@testing-library/user-event": "^7.1.2",
+    "@testing-library/dom": "^8.11.3",
+    "@testing-library/jest-dom": "^5.16.2",
+    "@testing-library/react": "^11.2.7",
+    "@testing-library/user-event": "^13.5.0",
     "@types/async": "^3.2.0",
-    "@types/classnames": "^2.2.8",
-    "@types/jest": "^24.0.17",
+    "@types/body-scroll-lock": "^3.1.0",
+    "@types/jest": "^27.4.1",
     "@types/lodash": "^4.14.149",
-    "@types/moment": "^2.13.0",
     "@types/node": "^12.12.7",
-    "@types/react": "^16.8.18",
-    "@types/react-dom": "^16.8.4",
-    "@types/react-test-renderer": "^16.9.0",
+    "@types/react": "^17.0.37",
+    "@types/react-dom": "^17.0.11",
+    "@types/react-test-renderer": "^17.0.1",
     "@types/styled-components": "^5.1.1",
     "@types/styled-system": "^5.1.10",
-    "@typescript-eslint/eslint-plugin": "^2.21.0",
-    "@typescript-eslint/parser": "^2.21.0",
+    "@typescript-eslint/eslint-plugin": "^5.6.0",
```

Comment:
Just a note that I had to upgrade to node v12.22.0 to use @typescript-eslint/eslint-plugin https://github.com/typescript-eslint/typescript-eslint/blob/main/package.json#L55

But my node version (v12.20.1) might be lagging behind because I haven't ran our react codebases in a while.

Reply:
Oooh, thanks. I realised we didn't really declare `engines` for the package.json too - maybe will add it in the future.